AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst

```diff
@@ -0,0 +1,103 @@
+Precondition
+============
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+The dome shutter is open and an unforeseeable condition arises, forcing an emergency dome closure.
+
+.. _Title-of-Procedure-Post-Condition:
+Post-Condition
+==============
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+- The emergency situation has been clearly notified to the relevant personnel and management.
+- Steps to eliminate or minimize the risk to the instruments and hardware are being implemented.
+- Eventually, the dome shutter is closed and all equipment is safe.
+
+.. _Title-of-Procedure-Procedure-Steps:
+Procedure Steps
+===============
+
+.. todo::
+   Please use the standard safety measures when visiting the dome enclosure. **Safety of personnel always goes first.**
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+In the event of an emergency dome closure, the dome shutter won't be able to be operated from the standard CSC controls. The AuxTel dome would need to be closed manually. In case of sudden inclement weather, be very cautious and start closing early.
+
+.. warning::
+    If you are dealing with a dome shutter emergency shutdown. Proceed calmly  to keep the equipment safe.
+
+.. _Title-of-Procedure-Critical-Step-1:
+
+Shutter Closure
+---------------
+
```

Comment:
I think there needs to be a first step, which is to communicate on #summit-announce that people are going to attempt to close the AuxTel dome via the emergency procedure.

Reply:
Added.